geonode/settings.py

```diff
@@ -2029,3 +2029,16 @@ def get_geonode_catalogue_service():
 # -- END Settings for MONITORING plugin
 
 CATALOG_METADATA_TEMPLATE = os.getenv("CATALOG_METADATA_TEMPLATE", "catalogue/full_metadata.xml")
+UI_DEFAULT_MANDATORY_FIELDS = [
+    'id_resource-title', 
+    'id_resource-abstract', 
+    'id_resource-language',
+    'id_resource-license',
+    'id_resource-regions',
+    'category_form', 
+    'id_resource-attribution', 
+    'id_resource-constraints_other',
+    'id_resource-data_quality_statement', 
+    'id_resource-restriction_code_type'
+]
+UI_REQUIRED_FIELDS  = ast.literal_eval(os.getenv('UI_REQUIRED_FIELDS ', '[]'))
```

Comment:
There is different behavior between documents/maps and layers.
If I set `UI_REQUIRED_FIELDS = ['id_resource-edition']`, I see it correctly set on the layer page:
![image](https://user-images.githubusercontent.com/51856725/116563185-14de2c00-a904-11eb-934f-e3d6496df090.png)
But on documents and maps, the label is not showed:
![image](https://user-images.githubusercontent.com/51856725/116563435-4fe05f80-a904-11eb-9c3b-776890b7f4c9.png)